test/specs/search/SearchService.spec.js

```diff
@@ -228,7 +228,9 @@ describe('ga_search_service', function() {
         '46°58′16.320030760136433" N  6°58′10.12802667678261" E',
         '46º58′16.320030760136433″ N  6º58′10.12802667678261″ E',
         '46° 58\' 16.320030760136433\'\' N 6° 58\' 10.12802667678261\'\' E',
-        // Quandrants are wrongi/missing, but we assume in Switzerland
+        // Skyguide special (zero padded)
+        '46°58\'16.320030760136433" N  006°58\'10.12802667678261" E',
```

Comment:
Add also a test with 1 zero.

Reply:
I've adde dit

Reply:
@oterral OK, I see. Switzerland spaning betwen 005°E and 011°E